Proiect_PAO/src/aplicatieBancara/Client/Client.java

```diff
@@ -0,0 +1,119 @@
+package aplicatieBancara.Client;
+
+import aplicatieBancara.Card.Card;
+import aplicatieBancara.Cont.Cont;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class Client {
+
+    private static int id = 0;
+    private final int idClient;
+    private String nume;
+    private String prenume;
+    private String CNP;
+    private Date dataNastere;
```

Comment:
it can be final

Reply:
M-am gandit ca numele si prenumele se pot modifica, de aceea am facut si metodele setNume si setPrenume. CNP si dataNastere le-am facut final.